pkg/migration/generate.go

```diff
@@ -117,6 +119,11 @@ func (h *Handler) createStorageCluster(
 }
 
 func parseCustomImageRegistry(pxImage string, componentImages []string) (string, bool) {
+	// Let's remove docker.io prefix, so we don't take docker.io as custom image registry.
```

Comment:
add this case to case 1 at line 128
```
// 1. portworx/oci-monitor:image or docker.io/portworx/oci-monitor:image
```
and handle it in case 1 as well
```
if len(imageParts) == 2 && imageParts[0] == "portworx"  || imageParts[0] == util.DefaultImageRegistry {
```

Reply:
there is no need to handle it, as the prefix is removed at this line already.